tensorflow_quantum/python/util.py

```diff
@@ -344,6 +345,86 @@ def _symbols_in_op(op):
         "tfq.util.get_supported_gates().")
 
 
+def is_expression_approx_eq(exp_1, exp_2, atol):
+    """Compare possibly symbolic expressions for approximate equality.
+
+    Coefficient based approximate equality.  If no symbol is present in
```

Comment:
Might want to provide an args and return list since you are passing in some non trivial things.

Reply:
Done